### PR TITLE
[Merged by Bors] - chore(Calculus/ParametricIntegralInterval): small clean-ups

### PR DESCRIPTION
**Pull request by grunweg**
- collect some very common variables
- use refine and \mapsto instead of refine' and => (both are preferred now)

----------

Easy to review. Best reviewed commit by commit (each says what it does). Preparatory cleaning for #10004.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/Calculus/ParametricIntegral.lean [hunk 1]
```diff
@@ -63,31 +63,32 @@ variable {α : Type*} [MeasurableSpace α] {μ : Measure α} {𝕜 : Type*} [IsR
   [NormedAddCommGroup E] [NormedSpace ℝ E] [NormedSpace 𝕜 E] [CompleteSpace E] {H : Type*}
   [NormedAddCommGroup H] [NormedSpace 𝕜 H]
 
+variable {bound : α → ℝ} {ε : ℝ} (ε_pos : 0 < ε)
```
Comment (urkud) [inline Mathlib/Analysis/Calculus/ParametricIntegral.lean:66]: Please don't move explicit arguments, especially assumptions, to `variable`s: this way `theorem` statements below miss important parts.

File: Mathlib/Analysis/Calculus/ParametricIntegral.lean [hunk 2]
```diff
@@ -63,31 +63,32 @@ variable {α : Type*} [MeasurableSpace α] {μ : Measure α} {𝕜 : Type*} [IsR
...
 slightly less general but usually more useful version. -/
 theorem hasFDerivAt_integral_of_dominated_loc_of_lip' {F : H → α → E} {F' : α → H →L[𝕜] E} {x₀ : H}
-    {bound : α → ℝ} {ε : ℝ} (ε_pos : 0 < ε)
     (hF_meas : ∀ x ∈ ball x₀ ε, AEStronglyMeasurable (F x) μ) (hF_int : Integrable (F x₀) μ)
     (hF'_meas : AEStronglyMeasurable F' μ)
     (h_lipsch : ∀ᵐ a ∂μ, ∀ x ∈ ball x₀ ε, ‖F x a - F x₀ a‖ ≤ bound a * ‖x - x₀‖)
     (bound_integrable : Integrable (bound : α → ℝ) μ)
-    (h_diff : ∀ᵐ a ∂μ, HasFDerivAt (fun x => F x a) (F' a) x₀) :
-    Integrable F' μ ∧ HasFDerivAt (fun x => ∫ a, F x a ∂μ) (∫ a, F' a ∂μ) x₀ := by
+    (h_diff : ∀ᵐ a ∂μ, HasFDerivAt (fun x ↦ F x a) (F' a) x₀) :
```
Comment (urkud) [inline Mathlib/Analysis/Calculus/ParametricIntegral.lean:78]: Another spelling (optional, both are accepted in Mathlib):
```suggestion
    (h_diff : ∀ᵐ a ∂μ, HasFDerivAt (F ⋅ a) (F' a) x₀) :
```

Comment (grunweg) [reply Mathlib/Analysis/Calculus/ParametricIntegral.lean:78]: This doesn't work yet: "expected token". (Presumably needs some notation open.) I've left it as-is.

Comment (urkud) [reply Mathlib/Analysis/Calculus/ParametricIntegral.lean:78]: I used a wrong middle dot. this one should work: `(F · a)`

**Comment by PatrickMassot** [comment]
bors merge

**Pull request closed without merging**